hammr/utils/hammr_utils.py

```diff
@@ -270,3 +270,29 @@ def extract_scan_id(image_uri):
         return int(match.group(1))
     else:
         return None
+
+def extract_appliance_id(image_uri):
+    match = re.match( r'users/[^/]+/appliances/([0-9]+)($|/)', image_uri)
+    if match:
+        return int(match.group(1))
+    else:
+        return None
+
+def check_mandatory_migration(migration):
+    return migration
+
+def check_mandatory_migrate(migration):
+    if not "name" in migration:
+        printer.out("no attribute [name] for [migration]", printer.ERROR)
+        return
+    if not "os" in migration:
+        printer.out("no attribute [os] for [migration]", printer.ERROR)
+        return
+    if not "source" in migration:
+        printer.out("no attribute [source] for [migration]", printer.ERROR)
+        return
+    if not "destination" in migration:
+        printer.out("no attribute [destination] for [migration]", printer.ERROR)
+        return
+    return migration
```

Comment:
the migration is never updated, why should we return it?

Reply:
ok I changed it